vnext/local-cli/runWindows/utils/build.js

```diff
@@ -73,55 +73,25 @@ async function nugetRestore(nugetPath, slnFile, verbose, msbuildVersion) {
 }
 
 async function restoreNuGetPackages(options, slnFile, verbose) {
-  let nugetPath =
-    options.nugetPath || path.join(os.tmpdir(), 'nuget.4.9.2.exe');
+  const nugetPath = path.join(os.tmpdir(), 'nuget.4.9.2.exe');
 
-  let downloadedNuget = false;
-  const ensureNugetSpinner = newSpinner('Locating NuGet executable');
   if (!(await existsAsync(nugetPath))) {
-    try {
-      nugetPath = execSync('where nuget')
-        .toString()
-        .trim();
-    } catch {}
-  }
-
-  if (!(await existsAsync(nugetPath))) {
-    await commandWithProgress(
-      ensureNugetSpinner,
-      'Downloading NuGet Binary',
-      'powershell',
-      `$progressPreference = [System.Management.Automation.ActionPreference]::SilentlyContinue; Invoke-WebRequest https://dist.nuget.org/win-x86-commandline/v4.9.2/nuget.exe -outfile ${nugetPath}`.split(
-        ' ',
-      ),
-      verbose,
+    const spinner = newSpinner('Downloading NuGet Binary');
+    await downloadFileWithRetry(
+      'https://dist.nuget.org/win-x86-commandline/v4.9.2/nuget.exe',
```

Comment:
We should support offline builds. This was made possible by checking whether nuget was already installed and using that. Can you restore that functionality?

Reply:
@asklar: Good point, but let's file an issue... We need to unblock our CI which is broken by the newly updated azure devops agents

Reply:
This previously grabbed local versions that were newer and totally broke. The first build definitely can't be offline, since the whole purpose of having NuGet is to download things.

Reply:
I think we should seriously reconsider how we're doing this in general, but would rather do the limited fix for something very broken and blocking check-in.

Reply:
Like, if anyone ever did a `choco install nuget` the previous logic makes it impossible to use `run-windows`, That's the image change that led to this.